core/pom.xml

```diff
@@ -34,6 +34,10 @@
                     <groupId>org.glassfish.jersey.connectors</groupId>
                     <artifactId>*</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>log4j</groupId>
```

Comment:
it's a common rule not to ship logging implementation with libraries. I have to idea why @docker-java does that.